README.md

```diff
@@ -36,7 +40,9 @@ Your answer here.
 What is the purpose of the `<head></head>` tag in HTML?
 
 ```text
-Your answer here.
+
+Purpose of <head> tag is to provide space for tags such as <title>, <meta> information, <style>, <script>, etc. 
+
```

Comment:
:+1:

Specifically, it contains information about the page (kind of like meta-data).